src/briefcase/integrations/android_sdk.py

```diff
@@ -307,12 +307,14 @@ def install(self):
             # Zip file no longer needed once unpacked.
             cmdline_tools_zip_path.unlink()
 
+            # fmt: off
             # Python zip unpacking ignores permission metadata.
             # On non-Windows, we manually fix permissions.
-            if self.tools.host_os != "Windows":
+            if self.tools.host_os != "Windows":  # pragma: no branch no-cover-if-is-windows
                 for binpath in (self.cmdline_tools_path / "bin").glob("*"):
                     if not self.tools.os.access(binpath, self.tools.os.X_OK):
                         binpath.chmod(0o755)
+            # fmt: on
 
```

Comment:
Black has `# fmt: skip`....but it only works if its the only comment on the line...

https://github.com/psf/black/issues/3450

Reply:
Clarifying - this is because the `# pragma` hits line length limits, so black tries to reformat, breaking coverage?

Reply:
yeah....i really wasn't a fan...

```python
            # Python zip unpacking ignores permission metadata.
            # On non-Windows, we manually fix permissions.
            if (
                self.tools.host_os != "Windows"
            ):  # pragma: no branch no-cover-if-is-windows
                for binpath in (self.cmdline_tools_path / "bin").glob("*"):
                    if not self.tools.os.access(binpath, self.tools.os.X_OK):
                        binpath.chmod(0o755)

```

Reply:
> breaking coverage?

To saw this part....let me check; I just really didn't like the re-format.

Reply:
This is what's necessary to get coverage on linux:

```python
            # Python zip unpacking ignores permission metadata.
            # On non-Windows, we manually fix permissions.
            if (  # pragma: no branch no-cover-if-is-windows
                self.tools.host_os != "Windows"
            ):
                for binpath in (self.cmdline_tools_path / "bin").glob("*"):
                    if not self.tools.os.access(binpath, self.tools.os.X_OK):
                        binpath.chmod(0o755)
```

Reply:
Ugh. All bad options, IMHO. I have a mild "we chose black, so we should live with it's sub-optimal choices" preference, but not enough to actually block a merge if you disagree.

Reply:
for posterity....getting coverage on all platforms is even worse....

```python
            # Python zip unpacking ignores permission metadata.
            # On non-Windows, we manually fix permissions.
            if (  # pragma: no branch
                self.tools.host_os != "Windows"
            ):  # pragma: no-cover-if-is-windows
                for binpath in (self.cmdline_tools_path / "bin").glob("*"):
                    if not self.tools.os.access(binpath, self.tools.os.X_OK):
                        binpath.chmod(0o755)
```